README.md

```diff
@@ -316,11 +314,97 @@ it('should use custom element', async() => {
 
 > **Important:** `defineCE` is not performant and shouldn't be used outside of test files.
 
-### Vdiff Testing
+## Running Tests
+
+Use the `d2l-test-runner` binary to execute a set of tests and report their results. It builds upon the robust [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/), while configuring it for Brightspace components and applications.
+
+### CLI and Configuration
+
+`d2l-test-runner` can be configured using CLI flags or an optional configuration file.
 
-Short for "visual diff" and also known as "visual regression" or "perceptual diff", vdiff testing involves taking snapshot images of the browser and comparing them against a known golden (or "baseline") image. The images are compared pixel-by-pixel and differences beyond a threshold will fail the test. Our vdiff testing leverages the [pixelmatch](https://github.com/mapbox/pixelmatch) library to perfom its comparison.
+#### Options
 
-Use the asynchronous `.to.be.golden()` Chai assertion to take a vdiff snapshot and compare it against its golden.
+| Name | Type | Default | Description |
+| :--- | :---: | :---: | :--- |
+| group | `String` | `'test'` | Name of the test group to run |
+| chrome | `Boolean` | `true` | Run tests in Chromium |
+| firefox | `Boolean` | `true` | Run tests in Firefox |
+| safari | `Boolean` | `true` | Run tests in Webkit |
```

Comment:
Technically doesn't default to `true` for vdiff, but probably not worth complicating the docs here.

Later Edit: And you do call it out in the `vdiff` section